src/libs/actions/Session/index.js

```diff
@@ -259,12 +265,28 @@ function signInWithShortLivedAuthToken(email, authToken) {
                 isLoading: false,
             },
         },
+        {
+            onyxMethod: CONST.ONYX.METHOD.MERGE,
+            key: ONYXKEYS.IS_TOKEN_VALID,
+            value: false,
+        },
     ];
 
     // If the user is transitioning to newDot from a different account on oldDot the credentials may be tied to
```

Comment:
should we update the comments here since we're also dealing with the situation of transitioning from web NewDot to desktop NewDot?